cmd/monoctl/create/cluster.go

```diff
@@ -72,6 +73,7 @@ func NewCreateClusterCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to read CA certificates from '%s': %s", caCertBundleFile, err)
 			}
+			caCertBundle = []byte(strings.TrimSpace(string(caCertBundle)))
```

Comment:
this is the only part related to this PR but meanwhile, I discovered that update kubeconfig will keep asking to choose a file even if one is already specified in m8Config (I could sware I tested this 1000 times but I think while c&p I broke the test by specifying custom file as mentioned below)